docs/visualization/plotting/plotting-1d-data.ipynb

```diff
@@ -6,6 +6,16 @@
    "source": [
     "# Plotting 1-D data\n",
     "\n",
+    "<div class=\"alert alert-warning\">\n",
+    "\n",
+    "*Deprecated*\n",
+    "\n",
+    "This particular guide uses the old plotting backend which is now deprecated and will be removed in the future.\n",
+    "See [Migrating to the New Backend](../plotting-overview.ipynb#Migrating-to-the-New-Backend) for general migration instructions.\n",
+    "And see the documentation of [Plopp](https://scipp.github.io/plopp/index.html) for details on plotting with the new backend.\n",
```

Comment:
This page and others in the folder: Is there a more concrete page we can/should link to? @nvaytet?